<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShaderViewerClass</class>
 <widget class="QMainWindow" name="ShaderViewerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>880</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ShaderViewer</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>851</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Tool</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuObject">
    <property name="title">
     <string>Object</string>
    </property>
    <widget class="QMenu" name="menu2DObject">
     <property name="title">
      <string>2DObject</string>
     </property>
     <addaction name="actionSqure"/>
    </widget>
    <widget class="QMenu" name="menu3DObject">
     <property name="title">
      <string>3DObject</string>
     </property>
    </widget>
    <addaction name="menu2DObject"/>
    <addaction name="menu3DObject"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuLoad">
     <property name="title">
      <string>Load</string>
     </property>
     <addaction name="actionModel"/>
     <addaction name="actionMaterial"/>
     <addaction name="actionScene"/>
     <addaction name="actionSkybox"/>
    </widget>
    <addaction name="actionOpen"/>
    <addaction name="menuLoad"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuWindow"/>
   <addaction name="menuTool"/>
   <addaction name="menuObject"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSqure">
   <property name="text">
    <string>Squre</string>
   </property>
  </action>
  <action name="actionCube">
   <property name="text">
    <string>Cube</string>
   </property>
  </action>
  <action name="actionMaterial">
   <property name="text">
    <string>Material</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionModel">
   <property name="text">
    <string>Model</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionScene">
   <property name="text">
    <string>Scene</string>
   </property>
  </action>
  <action name="actionSkybox">
   <property name="text">
    <string>Skybox</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ShaderViewer.qrc"/>
 </resources>
 <connections/>
</ui>
